tests/openqa/install/openqa_bootstrap.pm

```diff
@@ -25,6 +25,8 @@ sub run {
     }
 
     zypper_call('in openQA-bootstrap');
+    # aarch64 worker requires 'qemu-uefi-aarch64'
+    zypper_call('in qemu-uefi-aarch64') if check_var('ARCH', 'aarch64');
```

Comment:
Maybe we should rather fix the openqa-bootstrap script to install that version of qemu when being executed on aarch64.

Reply:
But it doesn't seem to be available on `x86_64` so we could add a check into openqa-bootstrap for this.

Reply:
`qemu-arm` already recommends `qemu-uefi-aarch64`, but not requires it, as it is available on aarch64 only so far.
`openqa-bootstrap` uses `--no-recommends` that is why it is not installed. 
`openqa-bootstrap` is arch independent, which is good IMO.

Reply:
@ggardet good catch... I think it would be better to specify install-recommends instead... @asdil12 wdyt?

Reply:
I also think that the install script should solve it, either by the *install-script* (not the test) explicitly installing depending on architecture or just using the recommends. If we just fix it in the tests then again the install script is no complete reference of how to bootstrap your openQA.

Reply:
I would rather not enable recommends as it installs too much useless stuff.
I will create a PR for openqa-review.

--> https://github.com/os-autoinst/openQA/pull/2080

Reply:
ok, I remove this part.